ontopy/ontodoc.py

```diff
@@ -288,6 +288,7 @@ def itemdoc(
             annotations.keys(), key=lambda key: order.get(key, key)
         ):
             for value in annotations[key]:
+                value = str(value)
```

Comment:
This could fail, I suppose, if the type of `value` cannot be cast to a `str` type.
You could instead wrap it in a `try/except` block, where you're catching `TypeError` and `ValueError` exceptions, re-raising them with an appropriate custom message?

Reply:
I discussed this with @jesper-friis , and we can't really see a situation in which this not enough. I think it is better to leave it as is, and update this if and when we find examples of situations in which a different handling is needed.

Reply:
As you wish. Casting to `str` is in general quite generous in this case, so it's fine I suppose. However, I'd argue it's good convention to consider possible edge cases and try to catch them if possible so exception messages are clear and helpful.